dask/bytes/local.py

```diff
@@ -15,6 +15,35 @@
 logger = logging.getLogger(__name__)
 
 
+def write_bytes(values, path, compression=None):
+    """ See dask.bytes.core.write_bytes for docstring """
+    nfiles = len(values)
+    if "{" not in path and "}" not in path:
+        try:
+            os.makedirs(path)
+        except Exception:  # exist_ok only available in py3
+            pass
+        if not os.path.exists(path) or not os.path.isdir(path):
+            raise ValueError("Path does not exist or is not a directory: %s",
+                             path)
+        path = os.path.join(path, "bytes-{:04d}")
+    paths = [path.format(i) for i in range(nfiles)]
```

Comment:
I'm somewhat tempted by the scheme with `*` and `name_function` that is already in there just to support backwards compatibility